cfme/fixtures/v2v.py

```diff
@@ -20,15 +21,23 @@ def v2v_providers(request, second_provider, provider):
 @pytest.fixture(scope='function')
 def host_creds(request, v2v_providers):
     """Add credentials to conversation host"""
-    provider = v2v_providers[1]
-    hosts = provider.hosts.all()
     try:
+        vmware_provider = v2v_providers[0]
+        hosts = vmware_provider.hosts.all()
+        for host in hosts:
+            host_data, = [data for data in vmware_provider.data['hosts']
+             if data['name'] == host.name]
+            host.update_credentials_rest(credentials=host_data['credentials'])
+
+        rhv_provider = v2v_providers[1]
+        hosts = rhv_provider.hosts.all()
         hosts = hosts if getattr(request, 'param', '') == 'multi-host' else hosts[0:1]
         for host in hosts:
-            host_data, = [data for data in provider.data['hosts'] if data['name'] == host.name]
+            host_data, = [data for data in rhv_provider.data['hosts'] if data['name'] == host.name]
             host.update_credentials_rest(credentials=host_data['credentials'])
     except Exception:
         # if above throws ValueError or TypeError or other exception, just skip the test
+        logger.exception("Exception when trying to add the host credentials.")
         pytest.skip("No data for hosts in providers, failed to retrieve hosts and add creds.")
     yield hosts
     for host in hosts:
```

Comment:
Should both the rhv and vmware hosts have the credentials removed?

Reply:
Good point, we should remove creds for both.